### PR TITLE
[Do Not Merge] Ironic Endpoint Keepalived changes

### PR DESCRIPTION
**Pull request by kashifest**
Co-authored-by: Xenwar <anwar.hassen@est.tech>

- Adding changes to make Ironic endpoint using keepalived to attach the correct VIP.
- Restructuring of kustomize folder

### CODE REVIEW

**Comment by kashifest** [comment]
Please do not merge. This PR is just for review

File: hack/keepalived-docker/manage-keepalived.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+export assigned_ip=172.22.0.2/24
+
+export interface=ironicendpoint
```
Comment (maelk) [inline hack/keepalived-docker/manage-keepalived.sh]: this needs to come from the configmap, as env variable

File: hack/keepalived-docker/manage-keepalived.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+while true;do 
+	/etc/init.d/keepalived start
+	ip a show $interface |grep "inet $assigned_ip"
+	if [ $? -gt 0 ]; then
+           echo "waiting for VIP to be assigned on interface $interface"
+	else
+           exit 0
+        fi
+	sleep 1
```
Comment (maelk) [inline hack/keepalived-docker/manage-keepalived.sh]: why looping the start command ? Is it expected to fail for a while and then succeed ?

Comment (kashifest) [reply hack/keepalived-docker/manage-keepalived.sh]: Yes it takes time for keepallived to run. Restart doesn't work in this scenario. Might be some bug in keepalived

File: hack/keepalived-docker/manage-keepalived.sh (hunk 1 above)
Comment (kashifest) [reply hack/keepalived-docker/manage-keepalived.sh]: Done

**Pull request merged**